### PR TITLE
Cleanup old images and manifests from kubernetes dev registry

### PR DESCRIPTION
**Pull request by yashgorana**
## Description

Registry got bloated to 160GB because of old manifests. A untagging old betas from CLI + scheduled purge on registry side fixes this issue.

## Affected Dependencies
List any dependencies that are required for this change.

## How has this been tested?
- Describe the tests that you ran to verify your changes.
- Provide instructions so we can reproduce.
- List any relevant details for your test configuration.

## Checklist
- [x] I have followed the [Contribution Guidelines](https://github.com/OpenMined/.github/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/OpenMined/.github/blob/master/CODE_OF_CONDUCT.md)
- [x] I have commented my code following the [OpenMined Styleguide](https://github.com/OpenMined/.github/blob/master/STYLEGUIDE.md)
- [x] I have labeled this PR with the relevant [Type labels](https://github.com/OpenMined/.github/labels?q=Type%3A)
- [x] My changes are covered by tests

### CODE REVIEW

**Review by rasswanth-s — APPROVED** [review]

**Comment by yashgorana** [comment]
Please do not merge this change right now

File: .github/workflows/cd-syft-dev.yml
Comment (rasswanth-s) [inline .github/workflows/cd-syft-dev.yml]: Q: Do we want to remove the commented section after a test run?

**Review by rasswanth-s — APPROVED** [review]
re-approving again

File: .github/workflows/cd-syft-dev.yml
Comment (yashgorana) [reply .github/workflows/cd-syft-dev.yml]: will remove once everything is working as expected

**Comment by yashgorana** [comment]
Closes https://github.com/OpenMined/Heartbeat/issues/984

**Comment by yashgorana** [comment]
Scheduled cleanup working as expected
image
image

File: .github/workflows/cd-syft-dev.yml
Comment (rasswanth-s) [inline .github/workflows/cd-syft-dev.yml:81]: After our testing and merge, could we also remove the secret and the Service Principal Account on Azure?

File: .github/workflows/cd-syft-dev.yml
Comment (rasswanth-s) [inline .github/workflows/cd-syft-dev.yml:182]: 👌

**Comment by rasswanth-s** [comment]
@yashgorana  are we good to merge this PR?

**Pull request merged**
